### PR TITLE
Add InvalidFormatExceptionTest JUnit tests

### LINKED ISSUE #299: Test if InvalidFormatException can return a message and a cause
InvalidFormatException should be able to return a message describing the error and a cause, if supplied when creating the exception.

### PR DESCRIPTION
**Pull request by matheril**
Ensures that message and cause can be retrieved from InvalidFormatException objects.
Fixes #299

### CODE REVIEW

**Review by warrencxw — APPROVED** [review]
Looks good to merge, but would be good if slight changes were made to add clarity.

File: src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        this.invalidFormatExceptionWithoutMessageWithoutCause = new InvalidFormatException();
+    }
+
+    @Test
+    public void getMessage_withMessage_returnsMessage() {
+        assertEquals("message", invalidFormatExceptionWithMessage.getMessage());
+    }
+
+    @Test
+    public void getCause_withMessage_doesNotReturnCause() {
```
[the remark this answers is not in the document]
Comment (matheril) [reply src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java]: Resolved

File: src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        String message = invalidFormatExceptionWithoutMessageWithoutCause.getMessage();
+        assertNull(message);
+    }
+
+    @Test
+    public void getCause_withoutMessageWithoutCause_doesNotReturnCause() {
+        Throwable cause = invalidFormatExceptionWithoutMessageWithoutCause.getCause();
+        assertNull(cause);
+    }
+}
```
Comment (froststein) [inline src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java:56]: Consider adding a new line at the end of the file

**Review by froststein — CHANGES_REQUESTED** [review]
Add a new line

File: src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java (hunk 2 above)
Comment (matheril) [reply src/test/java/seedu/splitlah/exceptions/InvalidFormatExceptionTest.java]: Resolved

**Review by froststein — APPROVED** [review]

**Pull request merged**